### PR TITLE
unhide system env (DY)LD_LIBRARY_PATH

### PR DESCRIPTION
**Pull request by bas524**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -68,7 +68,6 @@ Thumbs.db
 # VS generated files #
 ######################
 *.obj
-*.exe
```
Comment (aleks-f) [inline .gitignore]: why remove this?

Comment (bas524) [inline .gitignore]: because, it was set on line 7

**Comment by aleks-f** [comment]
This pull is now coming from two users, of which one (Alexander B) does not seem to be a github user. Also, please split these contributions into cohesive parts, adding update to 1.7.9p2 to a pull titled "unhide system env ..." is kindof messy (plus there are conflicts). Last but not least, in order for contributions to be accepted, all contributors must sign [CLA](https://cla-assistant.io/macchina-io/macchina.io)

**Comment by bas524** [comment]
sorry, both users are mine.. It's different options on different machines. How I can resolve conflicts? 
(I understand about different pulls)

**Comment by aleks-f** [comment]
> (I understand about different pulls)

I suggest to close this pull and create two new separate pulls.

I'm also, not sure about the poco upgrade ere, @obiltschnig will have to approve that (poco 1.8.0.1 is already released)

**Comment by bas524** [comment]
Ok, I agree. Can you close this pull request?

**Pull request closed without merging**